.github/workflows/unit-test.yml

```diff
@@ -42,13 +36,11 @@ jobs:
       - name: Unit tests
         run: |
           # TODO(legendecas): webpack https://stackoverflow.com/questions/69692842/error-message-error0308010cdigital-envelope-routinesunsupported
-          if [ "${{ matrix.node_version }}" = "18" ]; then
-            export NODE_OPTIONS=--openssl-legacy-provider
-          fi
+          export NODE_OPTIONS=--openssl-legacy-provider
           npm run test
       - name: Report Coverage
         run: npm run codecov
-        if: ${{ matrix.node_version == '14' }}
+        if: ${{ matrix.node_version == '18' }}
```

Comment:
Perhaps unrelated, but the uses of '16' below for browser-tests and webworker-tests could also be bumped to 18.

Likewise in "w3c-integration-test.yml" and "lint.yml".